frontend-new/src/pages/markdown.vue

```diff
@@ -12,32 +12,63 @@ const route = useRoute();
 // TODO Finish up and move to docs site
 const text = `
 The documentation of resources on hangar uses Markdown, which is a markup language that allows you to format text in a simple manner.
-You can learn how to use Markdown here: https://www.markdownguide.org/basic-syntax/
-
-The most basic formatting tools are:
-* ### Header: \`### Header\`, one # being the largest, five # being the smallest
-* **Bold text**: \`**Bold**\`
-* Italics text*: \`*Italics*\`
-* Add line breaks by adding two or more spaces at the end of a line
-* And more, like block quotes (\`> Text\`), lists (\`1. First item\`), unordered lists (\`* Item\`), and code blocks (\\\`Code\\\`)
-* Embedding an image: \`![Alt text](image link)\` ![Snowy](https://i.imgur.com/o1zcXsM.png =32x32)
-
-Additionally, Hangar provides the following features on top of basic Markdown syntax:
-* ~~Strikethrough text~~: \`~~This will be striked through~~\`
-* Links and e-mails are converted to clickable links automatically
-* Footnotes[^footnote]: \`text [^footnote]\`, and later \`[^footnote]: footnote text\`
-* Emojis \`:smile:\` :smile:
-* [Tables](https://github.com/vsch/flexmark-java/tree/master/flexmark-ext-tables/src/test/resources)
-
-Resizing images: \`![Alt text](Image URL =8x8)\`, in this case to size an image 8x8 pixels
-![Snowy](https://i.imgur.com/o1zcXsM.png =8x8)
-
-Embedding YouTube videos: \`@[YouTube](https://youtu.be/video-id)\`
-* @[YouTube](https://youtu.be/AG8VFyW61e0?t=3)
-
-Task lists: \`* [ ] First task\`, \`* [x] Completed task\`
-* [ ] Unfinished task
-* [x] Finished task
+
+## Basic Syntax
+Hangar supports all basic formatting options provided by the basic Markdown language.  
+A full guide on the basics of Markdown can be found here: https://www.markdownguide.org/basic-syntax
+
+## Additional Syntaxes
+The following features are provided on top of basic Markdown syntax:
+
+### Strikethrough
+Use \`~~text~~\` to make the text appear with a horizontal line through it.
+
+**Example:**  
+\`~~Strikethrough~~\`: ~~Strikethrough~~
+
+### URLs and E-Mails
+URLs and E-Mails will automatically be transformed into clickable links.
+
+**Example:**  
+\`https://papermc.io\`: https://papermc.io
+
+### Footnotes
+Use \`Text[^footnote]\` to create a footnote.  
+You also need to put \`[^footnote]: text\` somewhere in the document to make it work.
+
+**Example:**  
+\`Footnote[^footnote]\`: Footnote[^footnote]
+
+### Emojis
+You can use emojis by using the common Emoji-Syntax for it.
+
+**Example:**  
+\`:smile:\`: :smile:
+
+### Image resizing
+You can change the size of an image by appending \`=:widthx:height\` after the URL in an embedded image (Make sure to replace \`:width\` and \`:height\` with actual numbers).
```

Comment:
The =:widthx:height is confusing, even with the description after that

Reply:
What would be a better option then?

Reply:
I assume the lib/module/whatever for the markdown parsing has this `=<width>x<height>` with optional values?
Like that you could use either `=<width>x` or `=x<height>` and the missing option would be set to automatic?

In such a case would I probs use `=[width]x[height]` here to indicate optional values? Or should we just stick with <> throughout?